src/main/kotlin/net/maizegenetics/phgv2/cli/AlignAssemblies.kt

```diff
@@ -54,30 +55,50 @@ class AlignAssemblies : CliktCommand(help="Align assemblies using anchorwave") {
     private val myLogger = LogManager.getLogger(AlignAssemblies::class.java)
 
     val gff by option(help = "Full path to the reference gff file")
-        .required()
```

Comment:
Does the Clikt library not work correctly?

Reply:
@tcasstevens What you had originally worked fine.  I changed this so I could add the clikt parameters junit tests, for which I needed the .validate and the "require" then goes inside the .validate.